recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "adios2" %}
-{% set version = "2.10.1" %}
-{% set sha256 = "ce776f3a451994f4979c6bd6d946917a749290a37b7433c0254759b02695ad85" %}
-{% set build = 5 %}
+{% set version = "2.10.2" %}
+{% set sha256 = "14cf0bcd94772194bce0f2c0e74dba187965d1cffd12d45f801c32929158579e" %}
+{% set build = 0 %}
```

Comment:
@vicentebolea line 33f. (patch):
https://github.com/ornladios/ADIOS2/pull/4196

Was https://github.com/ornladios/ADIOS2/pull/4196 backported to ADIOS 2.10.2, too?
Cannot spot it in https://github.com/ornladios/ADIOS2/releases/tag/v2.10.2

Reply:
@ax3l that is correct, that PR has fallen into the cracks during this release, and I am unsure why it happened. I will make sure that it gets backported in the next release if we have one more patch release.